### DISCUSSION TITLE
Enable Columns breaking for mobile

### DISCUSSION THREAD

**Discussion by ThatGuySam** [WebDevStudios/nextjs-wordpress-starter#710] [Ideas]
The Columns component and block don't currently break into single columns for small/mobile screens. 

Perhaps this is on purpose but it should be a simple change [here](https://github.com/WebDevStudios/nextjs-wordpress-starter/blob/canary/frontend/components/atoms/Columns/Columns.module.css)

```
.columns {
  @apply grid gap-8 grid-cols-3;

  &.columns-1 {
    @apply grid-cols-1;
  }

  &.columns-2 {
    @apply grid-cols-2;
  }

 ...etc
```

Becomes

```
.columns {
  @apply grid gap-8 lg:grid-cols-3;

  &.columns-1 {
    @apply lg:grid-cols-1;
  }

  &.columns-2 {
    @apply lg:grid-cols-2;
  }

 ...etc
```

**Comment by itsamoreh** [comment]
Hello @ThatGuySam! Thanks for taking a look at the starter.

I was able to replicate this. I opened [an issue](https://github.com/WebDevStudios/nextjs-wordpress-starter/issues/885) so that we can get this fixed.
